### PR TITLE
Slack

### LINKED ISSUE #29: slack notification
implement slack notifications!

### PR DESCRIPTION
**Pull request by shankj3**
Slack integration, yo
closes #29

### CODE REVIEW

File: client/commandhelper/gitint_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ var goodgiturls = []struct {
 	acctRepo string
 }{
 	{"bitbucket ssh", []byte("git@bitbucket.org:level11consulting/ocelot.git"), "level11consulting/ocelot"},
-	{"bitbucket https", []byte("https://jessishank@github.com/shankj3/ocelot.git"), "level11consulting/ocelot"},
+	{"bitbucket https", []byte("https://jessishank@bitbucket.org/level11consulting/ocelot.git"), "level11consulting/ocelot"},
```
Comment (tjtelan) [inline client/commandhelper/gitint_test.go]: What's going on here? Is this intentional?

File: common/remote/slack/postwebhook.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/shankj3/ocelot/models/pb"
+	slack "github.com/shankj3/ocelot/models/slack/pb"
+)
+
+const ocelotIcon = "https://78.media.tumblr.com/avatar_06e2167f3e45_128.pnj"
```
Comment (tjtelan) [inline common/remote/slack/postwebhook.go]: Babou! How exotic!

File: docs/documentation/useage.md [hunk 3]
```diff
@@ -34,6 +34,8 @@ You can also run `ocelot creds vcs add` from the command line and it will prompt
 **todo** add repo walkthrough here  
 For credentials relating to items within your build, such as docker, gradle, or maven, you can upload using the `ocelot creds repo add` command  
 
+## Notify Credentials
```
Comment (tjtelan) [inline docs/documentation/useage.md]: Woo! Thanks for writing these too

File: client/commandhelper/gitint_test.go (hunk 1 above)
Comment (shankj3) [reply client/commandhelper/gitint_test.go]: yeah it was some copy paste bs from when i changed the paths, this shouldn't have originally changed from bitbucket to github

**Pull request merged**